pandas/tests/series/test_analytics.py

```diff
@@ -1759,6 +1759,48 @@ def test_value_counts_categorical_not_ordered(self):
         tm.assert_series_equal(idx.value_counts(normalize=True), exp)
 
 
+class TestMinCount():
+    @pytest.mark.parametrize("use_bottleneck", [True, False])
+    @pytest.mark.parametrize("method", [("sum", 0), ("prod", 1)])
```

Comment:
so for sum we *never* use bottleneck, I think for prod we DO use bottleneck. So maybe should just fix that here as well.

Reply:
In `_bn_ok_dtype`

```
        # GH 9422
        # further we also want to preserve NaN when all elements
        # are NaN, unlinke bottleneck/numpy which consider this
        # to be 0
        if name in ['nansum', 'nanprod']:
            return False

```

so we don't use bottleneck. And apparently bottleneck doesn't have a nanprod, at least in my version installed locally.